pallets/cash/src/chains.rs

```diff
@@ -73,12 +80,13 @@ impl ChainId {
             ChainId::Eth => ChainBlock::Eth(
                 runtime_interfaces::config_interface::get_eth_starport_parent_block(),
             ),
-            _ => panic!("XXX not implemented"),
+            ChainId::Reserved | ChainId::Dot => panic!("xxx not supported"),
```

Comment:
may save some time debugging later if the panic has the faulty chain it

Reply:
yeah. I also want to mostly remove panics, but some of these are hard to inhibit without having a concrete value to put in